src/steps/finalize.ts

```diff
@@ -0,0 +1,49 @@
+import { execSync } from 'node:child_process';
+import { unlinkSync } from 'node:fs';
+import path from 'node:path';
+
+import type { Options } from '../types/index.js';
+
+export default function finalize(options: Options): void {
```

Comment:
I don't think most codemods do this step automatically as everyone tends to have their own setup. But nice to see it worked for your use case 😄